numpy/ma/core.py

```diff
@@ -3118,6 +3118,46 @@ def __array_wrap__(self, obj, context=None):
 
         return result
 
+    def __array_function__(self, function, types, args, kwargs):
+        """
+        Wrap numpy functions that have a masked implementation.
+
+        Parameters
+        ----------
+        function : callable
+            Numpy function being called
+        types : iterable of classes
+            Classes that provide an ``__array_function__`` override. Can
+            in principle be used to interact with other classes. Below,
+            mostly passed on to `~numpy.ndarray`, which can only interact
+            with subclasses.
+        args : tuple
+            Positional arguments provided in the function call.
+        kwargs : dict
+            Keyword arguments provided in the function call.
+        """
+        ma_func = np.ma.__dict__.get(function.__name__, function)
+        # Known incompatible functions
+        #   copy: np.copy() has subok semantics, which we can't pass through
+        #   unique: using ma.view(ndarray) internally doesn't work
```

Comment:
I tried using `arr.view(np.ndarray)` within unique, but there are some expectations that a "masked" entity be returned. I would actually think that we want `ma.unique()` to ignore the mask altogether and that isn't one of the unique elements.

For now, this seems easiest to just skip.